### ISSUE TITLE
[SUGGESTION] can only use one badge

### ISSUE THREAD

**Issue by McKlean** [DOSE-420/qb-policebadge#2]
Sorry not really an issue but a request/suggestion

is it possible to have each badge linked to a job/rank so maybe
 job: police rank: 0 = ranger
 job: police rank: 1 = state
 job: police rank: 2 = sheriff
 job: police rank: 3 = lspd

and then maybe some for EMS?

**Comment by itsbull78** [comment]
did you find out how to use more then one badge

**Comment by Andyauk** [comment]
> Sorry not really an issue but a request/suggestion
> 
> is it possible to have each badge linked to a job/rank so maybe job: police rank: 0 = ranger job: police rank: 1 = state job: police rank: 2 = sheriff job: police rank: 3 = lspd
> 
> and then maybe some for EMS?

authorizedJobGrades = {0, 1, 2, 3, 4}

so you could break it up like 
authorizedJobGrades = {0}
authorizedJobGrades = {1}
authorizedJobGrades = {2}
authorizedJobGrades = {3}

in your PD Armory

**Comment by SulakAlNanna** [comment]
I think he means more of each department getting their own badge. there's more than one badge in there but all the badges have the same css so they all show LSPD regardless of department. I think its planned since there are multiple badge templates.

**Comment by ChaseBacon** [comment]
Has this been resolved? I have 3 depts and it only shows LSPD no matter the branch

**Comment by SulakAlNanna** [comment]
I don't think that was ever it's intention. You would need a to code a check for jobs then input the variations in the js, css, and html respectively. I think the intention was to do that in the beginning but never got around to it. I've seen some paid stuff that allow you to use multiple badges and variations but you typically can't add your own with those due to encoding.

**Comment by ChaseBacon** [comment]
> I don't think that was ever it's intention. You would need a to code a check for jobs then input the variations in the js, css, and html respectively. I think the intention was to do that in the beginning but never got around to it. I've seen some paid stuff that allow you to use multiple badges and variations but you typically can't add your own with those due to encoding.

fair enough   i mean it works great as is i won't lie if you only have LSPD
